### DISCUSSION TITLE
Adding a security policy

### DISCUSSION THREAD

**Discussion by lfons** [antvis/G6#6376] [General]
Thank you for the excellent work on the library. It would be very helpful to have a [security policy file](https://github.com/antvis/G6/security) that provides instructions on how to report a security vulnerability. Could you let us know if this is something that can be added, and if so, when we might expect it? Thank you!

**Comment by chandan637** [comment]
Great point @lfons,
Even I am unable to use the library as company's vulnerability tool is returning this (security.md) as a major issue. 
Requesting authors for quick action. Please add 2-3 contributors name and email as content for the security.md file.
